Sources/MFBBinding.m

```diff
@@ -419,6 +561,15 @@ - (void)mfb_bind:(NSString *)binding toObject:(id)observableController withKeyPa
     [bindingController bind];
 }
 
+- (void)mfb_unbindAll:(NSString *)binding
+{
```

Comment:
Also `MFBBindingStore` is not appropriate for setter bindings as we need to override existing bindings.